### DISCUSSION TITLE
1挖矿当前任务、进度不显示。2、挖币官网查询不到块

### DISCUSSION THREAD

**Discussion by Billylin1996** [starcoinorg/starcoin#2385] [Testnet]
我的账户 0x614b868671325d5862da198b1dc4b45f。

**Comment by Billylin1996** [comment]
![Uploading 360截图20210404232702142.jpg…]()

**Comment by Billylin1996** [comment]
![Uploading QQ图片20210404232950.png…]()
![Uploading QQ图片20210404232959.png…]()

**Comment by Billylin1996** [comment]
你们这个官网电脑打不开，手机上到时可以打开。

**Comment by jolestar** [comment]
上传的截图不对
